wiki/Gameplay/Input_device/Bemani_controller/en.md

```diff
@@ -0,0 +1,8 @@
+# Bemani Controller
+
+**Bemani Controller** is a term used to describe game controllers compatible with Bemani rythem games. 
```

Comment:
```suggestion
A **Bemani Controller** is a type of game controller or gamepad that is compatible with [Bemani](https://en.wikipedia.org/wiki/Bemani "Wikipedia")-like rythm games. 
```
Kinda difficult to describe considering that Bemani is a game developer. But something along these lines would help clarify that.